apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/LoggerOptions.cs

```diff
@@ -45,16 +52,19 @@ private LoggerOptions(LogLevel logLevel, MonitoredResource monitoredResource, Bu
         /// Create a new instance of <see cref="LoggerOptions"/>.
         /// </summary>
         /// <param name="logLevel">Optional, the minimum log level.  Defaults to <see cref="LogLevel.Information"/></param>
+        /// <param name="labels">Optional, custom labels to be added to log entries.</param>
         /// <param name="monitoredResource">Optional, the monitored resource.  The monitored resource will
         ///     be automatically detected if it is not set and will default to the global resource if the detection fails.
         ///     See: https://cloud.google.com/logging/docs/api/v2/resource-list </param>
         /// <param name="bufferOptions">Optional, the buffer options.  Defaults to a <see cref="BufferType.Timed"/></param>
         public static LoggerOptions Create(LogLevel logLevel = LogLevel.Information,
+            Dictionary<string, string> labels = null,
             MonitoredResource monitoredResource = null, BufferOptions bufferOptions = null)
         {
+            labels = labels ?? new Dictionary<string, string>();
```

Comment:
Are we calling this often? I wouldn't want to create an empty dictionary for no reason if we don't need to.